versions.json

```diff
@@ -236,12 +219,12 @@
     },
     "rdma_core": {
         "common": {
-            "branch": "stable-v48"
+            "branch": "v52.0"
         }
     },
     "pmix": {
         "common": {
-            "version": "4.2.9-1"
+            "version": "4.2.9"
```

Comment:
Why is the release omitted?

Reply:
was always `-1` on a lot of these. I can bring it back if it makes sense, but I was trying to align versioning with what is shown on the websites for ease. (the `-1` is moved to the code)

Reply:
I am thinking that if the release (`-1`) is moved to the code, there's a possibility that 2 files would need to be updated if the version is changed. i.e., it could be that the release is not always `-1`.